plugins/permission-common/src/types/api.ts

```diff
@@ -72,14 +72,44 @@ export type PermissionCondition<TParams extends unknown[] = unknown[]> = {
   params: TParams;
 };
 
+/**
+ * Utility type to represent an array with 1 or more elements.
+ * @internal
+ */
+export type NonEmptyArray<T> = [T, ...T[]];
```

Comment:
The api-reports builder doesn't seem to like this type 🤔 . I'm getting this error while building locally:

```
# Generating package API reports
## Processing plugins/permission-common
Error: Internal Error: Unable to follow symbol for "NonEmptyArray"

You have encountered a software defect. Please consider reporting the issue to the maintainers of this application.
```

Reply:
Looks like this is due to the `@internal` annotation. Marking as `@public` fixes the issue.